tests/validation/tests/v3_api/conftest.py

```diff
@@ -16,6 +16,57 @@ def pytest_unconfigure(config):
     if TEST_RBAC and CATTLE_TEST_URL:
         rbac_cleanup()
 
+
+def get_tls(certificate):
```

Comment:
There is the same method in `test_auth.py`
why not move it to common.py?

Reply:
Resolved. I used that function